generators/gulp/templates/gulpfile.js

```diff
@@ -12,6 +12,7 @@ var coveralls = require('gulp-coveralls');
 <% } -%>
 <% if (babel) { -%>
 var babel = require('gulp-babel');
+require('babel-core/register');
```

Comment:
I've updated with a comment and as it turns out the require to 'babel-core/register' is required as this is the published name of the module.